Clean Swift/Clean Swift/Service/NetworkService/NetworkService.swift

```diff
@@ -0,0 +1,34 @@
+// NetworkService.swift
+// Copyright © Alexandr T. All rights reserved.
+
+import Foundation
+
+/// Сетевой слой
+final class NetworkService: NetworkServiceProtocol {
+    // MARK: - Private Constants
+
+    private enum Constants {
+        static let spaceUrlString = "https://go-apod.herokuapp.com/apod"
+    }
+
+    // MARK: - Public Methods
+
+    func fetchSpace(completion: @escaping ((Result<Astronomy, Error>) -> Void)) {
```

Comment:
```suggestion
    func fetchSpace(completion: @escaping ((Result<AstronomyModel.InitForm.Response, Error>) -> Void)) {
```
насколько я понял, данные уже передаются везде через модель